services/consensusalgo/benchmarkconsensus/non_leader.go

```diff
@@ -84,7 +84,8 @@ func (s *service) nonLeaderCommitAndReply(ctx context.Context, blockPair *protoc
 	status := (&gossipmessages.BenchmarkConsensusStatusBuilder{
 		LastCommittedBlockHeight: lastCommittedBlockHeight,
 	}).Build()
-	sig, err := digest.SignAsNode(s.config.NodePrivateKey(), status.Raw())
+	sig, err := kms.GetSigner(s.config).Sign(status.Raw())
```

Comment:
we need to understand the implications of failure here as we now have to deal with HTTP errors

Reply:
remove call to GetSigner here

Reply:
inject instantiated signer from top